web/src/Update.elm

```diff
@@ -133,6 +133,21 @@ update msg model =
             in
             { model | democracies = Dict.insert democId setDelegateState model.democracies } ! []
 
+        Debug str ->
+            let
+                currentLog =
+                    Dict.get lSKeys.debugLog model.localStorage ? ""
+
+                -- TODO: model.now does't update in real time.
+                -- TODO: convert model.now from float to ISO format
```

Comment:
better to keep as JS Date obj or as a timestamp and convert to ISO as needed, that way you can use it as an index in a Dict and directly compare them.

Reply:
Ahh, I see that was the intention anyway. All g. Timestamps are perfectly okay for this atm.